CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ option(WITH_COMPS "Build with comps groups and environments support" ON)
 option(WITH_MODULEMD "Build with modulemd modules support" ON)
 option(WITH_ZCHUNK "Build with zchunk delta compression support" ON)
 option(WITH_SYSTEMD "Build with systemd and D-Bus features" ON)
+option(WITH_APPSTREAM "Build with appstream support" ON)
```

Comment:
I think now we don't want here the `WITH_APPSTREAM` switch and rather move it inside the `appstream/CMakeLists.txt`.

Reply:
Similarly as it is handled for the `actions` plugin.